test/pee.test.cjs

```diff
@@ -0,0 +1,27 @@
+const { leak } = require('../pee.cjs');
+
+const amountOfPee = 10
+
+test('it can pee', (andMakesSureThereIsNoLastDrop) => {
+    console.log('Testing commonjs')
+    let initiallyUsed = process.memoryUsage().heapUsed / 1024 / 1024;
+    console.log(`The script uses approximately ${Math.round(initiallyUsed * 100) / 100} MB before Peeing.`);
+    leak(amountOfPee);
+    let used = process.memoryUsage().heapUsed / 1024 / 1024;
+    let peeDifferential = used - initiallyUsed
+    console.log(`The script uses approximately ${Math.round(used * 100) / 100} MB after Peeing.`);
+    // TODO: this fails
+    expect(peeDifferential >= amountOfPee).toBeTruthy()
```

Comment:
in this case when adding a 
```javascript
console.log(initiallyUsed, used, peeDifferential)
```
right above this expect, i usually end up seeing that the `peeDifferential` is about half of the expected `10` value